config/routes.rb

```diff
@@ -6,7 +6,7 @@
   # Set home pages:
   root :to => 'home#index'
   get 'home' => 'home#dashboard', :as => :dashboard
-
+  get 'videos' => 'videos#index'
```

Comment:
I built a basic AJAX system. I'll push when I've finished making it a teeny bit prettier.